File: Hemo backend/model/Blood.js
const mongoose = require('mongoose')

const bloodSchema = new mongoose.Schema({
    user:{
        type: String,
        required: true
    },
    status: {
        type: Boolean,
        default: true
    },
    created: {
        type: Date,
        default: Date.now()
    },
    latitude:{
        type: Number,
        required: true
    },
    longitude:{
        type: Number,
        required: true
    },
    location:{
        type: String,
        required: true
    },
    blood: {
        type: String,
        required: true
    },
    quantity:{
        type: Number,
        required: true
    }
})

const Blood = mongoose.model('Blood', bloodSchema);
module.exports=Blood;

File: Blood App/app/src/main/java/com/dev334/blood/model/Blood.java
package com.dev334.blood.model;

public class Blood {
    private String user;
    private Double latitude;
    private Double longitude;
    private String location;
    private Integer quantity;
    private String blood;

    Blood(){
        //empty constructor
    }

    public Blood(String user, Double latitude, Double longitude, String location, Integer quantity, String blood) {
        this.user = user;
        this.latitude = latitude;
        this.longitude = longitude;
        this.location = location;
        this.quantity = quantity;
        this.blood = blood;
    }

    public String getUser() {
        return user;
    }

    public Double getLatitude() {
        return latitude;
    }

    public Double getLongitude() {
        return longitude;
    }

    public String getLocation() {
        return location;
    }

    public Integer getQuantity() {
        return quantity;
    }

    public String getBlood() {
        return blood;
    }
}